sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/AfterWatermark.java

```diff
@@ -64,7 +64,7 @@
 @Experimental(Experimental.Kind.TRIGGER)
 public class AfterWatermark {
 
-  private static final String TO_STRING = "AfterWatermark.pastEndOfWindow()";
+  static final String TO_STRING = "AfterWatermark.pastEndOfWindow()";
```

Comment:
Since you're treating DefaultTrigger as default, you shouldn't need this to be visible, right?